### ISSUE TITLE
MCP lock error in synchronizing

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2773]
What is an MCP lock error? Alas, it happened quickly and when I was meeting with Alyssa for IBASA and so we got over it too quickly - I didn't make a screenshot. She initiated synchronization. We were both in Kiosk, and I refreshed my browser very shortly after she started synchronization. She then saw an MCP lock error message. When my browser started again I could get to the same error, though it didn't pop up for me. I then re-started synchronization and it worked fine, but I didn't like that error. Can it have had something to do with me refreshing my browser too fast, even though the command to synchronize came from her? Or is it something about her privileges? I don't want that happening to her in the field.

**Comment by urapadmin** [comment]
no, it has something to do with concurrent requests. Kiosk tries to make sure that two people cannot do initiate things at the same time that mustn't run parallel. On Beset we see this error sometimes as a result. Not sure why but it is harmless and can be treated like any other hiccup.

Reminds us yet again that Beset is not built to be used during excavation.

But I leave this open to have a look at the logs. It has come up too often recently.
